app/src/main/java/atlant/moviesapp/presenters/MovieListPresenter.java

```diff
@@ -0,0 +1,73 @@
+package atlant.moviesapp.presenters;
+
+import android.util.Log;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import atlant.moviesapp.model.Movie;
+import atlant.moviesapp.model.MovieGenre;
+import atlant.moviesapp.model.MoviesResponse;
+import atlant.moviesapp.rest.ApiClient;
+import atlant.moviesapp.rest.ApiInterface;
+import atlant.moviesapp.views.MovieListView;
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
+
+/**
+ * Created by Korisnik on 12.04.2017..
+ */
+
+public class MovieListPresenter {
+
+    private MovieListView view;
+    private static final int MOST_POPULAR = 0;
+    private static final int LATEST = 1;
+    private static final int HIGHEST_RATED = 2;
+
+
+    private final static String API_KEY = "e87991d039ddbe98de1958bf8a8b5148";
```

Comment:
Hide the API_KEY, it is a bad practice sharing it on github, 
Consider set the final String const API_KEY as a buildConfigField.
Put the buildConfigField, in the build.gradle android tag, and set the actual value of the key in gradle.properties which should be included in the .gitignore